.editorconfig

```diff
@@ -12,10 +12,9 @@ end_of_line = lf
 max_line_length = 120
 
 [*.{kt,kts}]
-ij_kotlin_imports_layout = *,^
+ij_kotlin_imports_layout = *, ^
```

Comment:
Question: Was this an intentional change? If so, what does it change?

Reply:
Hmm, it's Android Studio. When I have the file opened and it's saved, it is adjusted like that.